src/h5web/providers/my-utils.ts

```diff
@@ -86,6 +92,28 @@ export function makeMyDatatype<T extends HDF5Type>(
   };
 }
 
+export function makeMyExternalLink(
+  name: string,
+  file: string,
+  h5path: string,
+  opts: Omit<EntityOpts, 'id'> = {}
```

Comment:
Isn't it just `{attributes}` ?

Reply:
I have a fix coming for this in my next PR. Technically, a link doesn't have attributes, so this function can be simplified.